README.md

```diff
@@ -42,14 +42,13 @@ The sample code for Tax Service includes the following Apex classes :
 
 ### Error Handling
 
-There are two types of errors that can surface from the reference implementations: user errors (which the shopper can see and correct) and admin errors (which the shopper can’t fix).
+In order to propagate an error message to the shopper use [CartValidationOutput](https://developer.salesforce.com/docs/commerce/salesforce-commerce/guide/CartValidationOutput.html).
+Throwing an Exception from any Orchestrator or Calculator would result into CartValidationOutput created on the Cart with generic error message based on Calculator type, telling shopper to contact their admin.
```

Comment:
I'm not sure it ever makes sense to "tell a shopper to contact their admin."

A typical shopper doesn't know or care about the site's administrator, and likely has no idea how to contact the admin. 

Faced with an error that requests them to contact their admin, shoppers would likely give up and leave the site.

Reply:
Thanks @pdisabito !
This is default [label](https://sourcegraph.soma.salesforce.com/perforce.soma.salesforce.com/app/main/core/-/blob/shared-labels/java/resources/sfdc/i18n/shared_core_ui_labels/ui-commerce.xml?L182) we use since B2B Aura checkout for this kind of issues.

I'm going to remove "contact admin" part from this PR to stop further bleeding of this recommendation.